aea/mail/base.proto

```diff
@@ -1,6 +1,24 @@
 syntax = "proto3";
 
-package fetch.aea;
+package fetchai.aea;
+
+import "google/protobuf/struct.proto";
+
+
+message DialogueMessage {
+    int32 message_id = 1;
+    string dialogue_starter_reference = 2;
+    string dialogue_responder_reference = 3;
+    int32 target = 4;
+    bytes content = 5;
+}
+
+message Message {
+    oneof message {
+        google.protobuf.Struct body = 1;
```

Comment:
How useful is `google.protobuf.Struct` in comparison to a plain `string` field containing a JSON object serialization?